queries.sql

```diff
@@ -33,41 +35,63 @@ SELECT * FROM albums ORDER BY title DESC;
 SELECT * FROM albums WHERE title LIKE 'A%' ORDER BY title;
 
 -- "invoices" table
+SELECT invoices;
 -- Remember: run "\d+ invoices" to see the structure of the "invoices" table
 
 -- Every invoice
+SELECT * FROM invoices;
 
 -- Every invoice ordered by total invoice amount ("total")
+SELECT * FROM invoices ORDER BY total;
+-- SELECT total FROM invoices ORDER BY ?
+-- SELECT total would get all the totals
 
 -- Every invoice with a total greater than 10
+SELECT * FROM invoices WHERE total>10;
+-- WHERE is wrong?
 
 -- The 10 least expensive invoices
 -- Remember: ORDER BY orders from lowest-to-highest by default
 SELECT * FROM invoices ORDER BY total LIMIT 10;
 
 -- The 10 most expensive invoices
+SELECT * FROM invoices ORDER BY total DESC;
 
 -- The 15 most recent invoices
+SELECT * FROM invoices ORDER BY invoice_date LIMIT 15;
 
 -- The 15 oldest invoices
+SELECT * FROM invoices ORDER BY invoice_date DESC LIMIT 10;
 
 -- The 10 most expensive invoices from the US
+SELECT * FROM invoices ORDER BY billing_country, total DESC LIMIT 10 WHERE billing_country = 'USA';
+--sort by country then total
 
 -- The 10 least expensive invoices from the US
+SELECT * FROM invoices ORDER BY billing_country, total LIMIT 10 WHERE billing_country = 'USA';
+--SELECT * FROM invoices WHERE billing_country = 'US' ORDER BY total LIMIT 10;
+-- unsure which way is correct, but AM sure I dont know the correct order to place the queries
+
 
 -- The 10 most expensive invoices from outside the US
+SELECT * FROM invoices ORDER BY total WHERE billing_country != 'USA';
+-- I know I ordered this one differently than the above, im unsure which is correct but this one seems the best?
 -- Hint: If "=" means equal, use "!=" to mean "not equal"
+-- does <> work for not equal as well
 
 -- Every invoice from Chicago, IL
 SELECT * FROM invoices WHERE billing_city = 'Chicago' AND billing_state = 'IL' AND billing_country='USA';
 
 -- A list of all the invoices worth more than $5.00 from Chicago, IL
+SELECT * FROM invoices WHERE billing_city = 'Chicago' AND billing_state = 'IL' AND billing_country='USA' AND total>10;
 
 -- The billing addresses of the 5 most valuable invoices from Mountain View CA
 -- Gotta reward those big spenders!
+SELECT * FROM invoces ORDER BY total DESC LIMIT 5 WHERE billing_city = 'Mountain View' AND billing_state = 'CA';
 
 -- A list of the 10 most valuable invoices made before January 1st, 2010
 -- Hint: Dates are formatted like 'YYYY-MM-DD' and you can compare them using '<', '>', '<=' and '>='
+SELECT * FROM invoices ORDER BY total DESC LIMIT 10 WHERE invoice_date < 2010-01-01;
```

Comment:
It's going to be sort of hopeless so long as the clauses are in the wrong order. You shouldn't be "guessing"; run the queries in the `psql` shell to see the error message and try to debug.

There are working examples for all the clauses already in this file. Copy the relative order of the clauses.